deploy/operator/setup_assisted_operator.sh

```diff
@@ -56,6 +56,15 @@ EOF
     if [ -n "${SERVICE_BASE_URL:-}" ]; then
 cat <<EOF
   SERVICE_BASE_URL: '${SERVICE_BASE_URL}'
+EOF
+    fi
+
+    IMAGES=(${AGENT_IMAGE} ${CONTROLLER_IMAGE} ${INSTALLER_IMAGE} ${DATABASE_IMAGE})
+    CI_REGISTRIES=$(for image in ${IMAGES}; do echo ${image} | cut -d'/' -f1; done | sort -u | paste -sd "," -)
```

Comment:
This logic already exists as part of assisted-setup logic.
https://github.com/openshift/release/blob/d9cde1bcc3b2ec8bbbb62eeb818acbf4e2d062f1/ci-operator/step-registry/baremetalds/assisted/setup/baremetalds-assisted-setup-commands.sh#L91-L94

I'm not sure that we want this logic to be here, since the images that are being used aren't necessarily public.

Reply:
ok, fixed